scipy/stats/stats.py

```diff
@@ -2,25 +2,22 @@
 # Use the `scipy.stats` namespace for importing the functions
 # included below.
 
-import warnings
-from . import _stats_py
+from scipy._lib.deprecation import _sub_module_deprecation
 
 
 __all__ = [  # noqa: F822
     'find_repeats', 'gmean', 'hmean', 'pmean', 'mode', 'tmean', 'tvar',
     'tmin', 'tmax', 'tstd', 'tsem', 'moment',
     'skew', 'kurtosis', 'describe', 'skewtest', 'kurtosistest',
-    'normaltest', 'jarque_bera', 'itemfreq',
```

Comment:
Can you remind me why it's OK to remove all of these? I'm thinking that the result objects were never supposed to be public and have never been documented? Some of the others were already deprecated and removed?

Reply:
Simply that they don't exist in SciPy any more but were left in the `__all__`.

Reply:
: ) That makes it easy.